README.md

```diff
@@ -237,52 +158,41 @@ Example:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/Cluab/stock-market/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- SUPPORT -->
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+Give a ⭐️ if you like this project!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+I would like to thank Microverse for project inspiration.
+Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
```

Comment:
- [Optional] Good job adding the acknowledgment. Yet you could be more specific and use a separate line to acknowledge both `Microverse` and `[Nelson Sakwa`